doc/conf.py

```diff
@@ -293,7 +293,6 @@
         "of Google Inc."
     ],
     "toc_overview": True,
-    "navbar_active_link": 3,
```

Comment:
Was this an issue with the docs build or something?

Reply:
Yeah, sphinx had an error (same as the one in Pennylane), so I was advised to make this change to fix it 😄.